releases/release-6.1.3.md

```diff
@@ -66,6 +66,7 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.1/quick-start-with-
     + TiCDC
 
         - Fix data loss occurred in the scenario of executing DDL statements first and then pausing and resuming the changefeed [#7682](https://github.com/pingcap/tiflow/issues/7682) @[asddongmen](https://github.com/asddongmen)
+        - Fix the issue that the sink module gets stuck if the downstream network is unavailable [#7706](https://github.com/pingcap/tiflow/issues/7706) @[hicqu](https://github.com/hicqu)
```

Comment:
```suggestion
        - Fix the issue that the sink component gets stuck if the downstream network is unavailable [#7706](https://github.com/pingcap/tiflow/issues/7706) @[hicqu](https://github.com/hicqu)
```

Keep it consistent with [TiCDC Overview](https://docs.pingcap.com/tidb/dev/ticdc-overview#:~:text=Currently%2C%20the%20TiCDC%20sink%20component%20supports%20replicating%20data%20to%20the%20following%20downstream%20platforms%3A)